packages/azure-http-specs/specs/azure/core/page/mockapi.ts

```diff
@@ -71,10 +71,6 @@ Scenarios.Azure_Core_Page_withParameterizedNextLink = passOnSuccess([
     handler: (req: MockRequest) => {
       return {
         status: 200,
-        body: json({
-          value: [{ id: 1, name: "User1" }],
-          nextLink: `${req.baseUrl}/azure/core/page/with-parameterized-next-link/second-page?select=${req.query.select}`,
-        }),
       };
     },
```

Comment:
@tadelesh Why remove the response? I assume you want to remove the nextLink in response, not handler?